vault-init-eso.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+
+enable_eso() {
+    local eso_service_account_namespace
+    local eso_service_account_name
+    local eso_pod_name
+
+    export SA_JWT_TOKEN=$(kubectl -n vault get secret auth-sa-token -o jsonpath="{.data.token}" | base64 --decode; echo)
```

Comment:
you have an option to change vault namespace in helm, but have it hardcoded here